src/app_service/service/message/service.nim

```diff
@@ -596,9 +608,9 @@ QtObject:
     except Exception as e:
       error "error: ", procName="onAsyncLoadCommunityMemberAllMessages", errName = e.name, errDesription = e.msg
 
-  proc addReaction*(self: Service, chatId: string, messageId: string, emojiId: int) =
+  proc addReaction*(self: Service, chatId: string, messageId: string, emojiId: int, emoji: string) =
```

Comment:
Where is currently the line between `emoji`/`emojiID`? I guess the intention was between QML / Nim, because we don't have the UI yet, so Nim would be the conversion layer.

But I can see `emojiId` on all levels in Nim 🤔
It would be nice to have it in a single place, closest to the QML layer, like `view.nim`.

Reply:
Yeah, the end goal is to remove `emojiId` from everywhere. It will no longer be needed, but with how it's currently implemented, I can't remove it (it's used in the model for example).

One I hook everything, I'll do a massive cleanup/refactor and we'll get rid of `emojiId`